@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -144,6 +145,7 @@ const tempScreenPosition = new THREE.Vector2();
 const tempScreenPoints: THREE.Vector2[] = [];
 const tempPoiScreenPosition = new THREE.Vector2();
 const tempTextOffset = new THREE.Vector2();
+const tmpTextBufferCreationParams: TextBufferCreationParameters = {};
```

Comment:
Could you use TempParams (passed to addPointLabel) instead?

Reply:
I'm planning to get rid of that TempParams structure, it contains parameter structures with no relation to each other, some of them are unused. I'll remove it in my next changes.